### ISSUE TITLE
Final result integration

### ISSUE THREAD

**Issue by chickenflyingtosky** [daiqile96/OTTERS#6]
Hi, I'm new to population genetics and I'm currently using OTTERS for TWAS analysis and I have obtained the following result files: lassosum.txt, P0.001.txt, P0.05.txt, PRScs.txt, and SDPR.txt.

My first question is whether I should use the p-values of a single gene from these five files as input for the ACAT package in order to obtain the final p-value?

Secondly, I perform GWAS of 87 samples and get a not very satisfactory results, then I caculate eQTL of 49 samples and finally used the GWAS summary and eQTL sammary as the input of OTTERS,  the results are not very satisfactory(few gene pvalues less than 0.01). If I increase the eQTL sample size to around 80, will the results be improved? 

The species I am studying is wheat, which has three subgenomes and a genome size of 14G. Therefore, I am wondering if OTTERS is suitable for TWAS analysis in wheat.

I'm looking forward to your response. Thank you very much!

**Comment by daiqile96** [comment]
Hi,

Thank you for using OTTERS.

Regarding your first question, yes, that is the correct method for obtaining the final OTTERS p-value. Additionally, we have recently discovered that SDPR is not particularly sensitive to the reference panel. Therefore, if you notice that many genes with significant ACAT p-values are predominantly driven by SDPR, I recommend using only the p-values from lassosum.txt, P0.001.txt, P0.05.txt, PRScs.txt, and conducting an ACAT analysis on them.

For your second question, I do believe that increasing the eQTL sample size could _potentially_ enhance statistical power and lead to the identification of more significant genes. However, I have some concerns about the sample sizes of the eQTL and GWAS analyses. Is ~80 considered a relatively large sample size for conducting GWAS in wheat?

Regarding the suitability of OTTERS for TWAS analysis in wheat, I am curious about the average number of eQTLs you have for each gene. I think having three subgenomes and a genome size of 14G should be fine. But, as I have limited knowledge about wheat genetics, I will consult with others and get back to you later.

Best,
Qile

**Comment by chickenflyingtosky** [comment]
Hi,
Thank you very much for your reply.
I think the biggest reason is the smaller sample size. 80 samples is still a very little sample size for GWAS, and the result is not good, so i'm trying to use eqtl to make the result better, but it seems not a good way. 
 I used matrixeqtl to caculate eqtl with a p value with threshold 0.0025, the result eQTLs for each gene is  ~14000. Finally I  get 10 genes (107891 genes all) with p value < 0.0001, So I'm wondering what extent can I ues this result?

**Comment by chickenflyingtosky** [comment]
Hi,
Thank you very much for your reply.
I think the biggest reason is the smaller sample size. 80 samples is still a very little sample size for GWAS, and the result is not good, so i'm trying to use eqtl to make the result better, but it seems not a good way.
I used matrixeqtl to caculate eqtl with a p value with threshold 0.0025, the result eQTLs for each gene is ~14000.

**Comment by daiqile96** [comment]
Yes, and 80 samples are also a small sample size for the eQTL study. Since OTTERS only requires summary-level eQTL and GWAS data, one possible way to increase the sample size is to conduct eQTL and GWAS meta-analysis, and then apply OTTERS to the summary-level statistics generated from the meta-analyses.

Another consideration regarding whether OTTERS is suitable for TWAS in wheat is the kinship between the samples. Similar to GWAS, TWAS also assumes that all subjects are not closely related.

**Comment by jwhbaby123** [comment]
Hi,"I have three questions while using OTTERS:
Why do some genes have NA values in the results of a few of the five models?
Why does ACAT still provide a p-value result when all five models have NA values?
After correcting the p-values of the GWAS summary data, why do genes that originally had calculable results become NA values?"
